ant/src/org/jetbrains/kotlin/ant/Kotlin2JvmTask.kt

```diff
@@ -73,4 +81,42 @@ class Kotlin2JvmTask : KotlinCompilerBaseTask() {
         if (noReflect) args.add("-no-reflect")
         if (includeRuntime) args.add("-include-runtime")
     }
+
+    override fun execute() {
```

Comment:
It seems that overriding `execute` resulted in the fork mode not having all the logic that was there, such as:
* filling different compiler arguments (`-nowarn`, `-verbose`, `-include-runtime`, `-module-name`, etc.)
* running the compiler with `execFullPathsInMessages` as opposed to the default mode when paths in compiler messages are relative
* logging what's being compiled
* throwing `BuildException` on compilation error

Reply:
Thanks for your view and pointed out those mistakes. 
I have made some modifications about you mentioned above. But I'm confused about 

> running the compiler with execFullPathsInMessages as opposed to the default mode when paths in compiler messages are relative

Did you mean MessageRenderer? I ran some tests, but I didn't see difference between `RelativePath MessageRenderer `and `FullPath MessageRenderer`. Is it should be MessageRenderer in fork mode?

Besides, I find the prefix "[kotlinc]" missing in my `build.log.expected`, trying to find out a solution about it.

Reply:
> Besides, I find the prefix "[kotlinc]" missing in my build.log.expected, trying to find out a solution about it.

This output format problem is solved.

Reply:
Thanks! I meant that the compiler invoked from the Ant task uses full paths in the messages by default, and it's a bit weird that enabling `fork` changes that to relative paths.

However, it's not a big problem at all, and seeing that there's no API in the compiler for running it _externally_ with full paths in messages, I suppose we can leave it as is, with relative paths in the fork mode.